runtime/rococo/src/lib.rs

```diff
@@ -1552,7 +1552,6 @@ pub mod migrations {
 		parachains_configuration::migration::v7::MigrateToV7<Runtime>,
 		assigned_slots::migration::v1::VersionCheckedMigrateToV1<Runtime>,
 		parachains_scheduler::migration::v1::MigrateToV1<Runtime>,
-		parachains_configuration::migration::v7::MigrateToV7<Runtime>,
```

Comment:
I think we should keep this for now. I'm not sure if the migration is released everywhere.

Reply:
I removed it because it had been added twice to the Unreleased struct. It's still there a couple of lines up the struct.

Reply:
Oh I see, sorry.